src/api/store/community.py

```diff
@@ -183,29 +182,31 @@ def create_community(self,context: Context, args) -> (dict, MassEnergizeAPIError
           comm_admin.save()
 
       #also clone all global actions for this community
-      global_actions = Action.objects.filter(is_global=True)
-      for action_to_copy in global_actions:
-        action_to_copy_id = action_to_copy.id
-        old_tags = action_to_copy.tags.all()
-        old_vendors = action_to_copy.vendors.all()
-        new_action: Action = action_to_copy
-        new_action.pk = None
-        new_action.is_published = False
-        new_action.is_global = False
+      #global_actions = Action.objects.filter(is_global=True)
```

Comment:
could you explain why you are removing all this code?

Reply:
Thank you for asking! What I saw was that in creating a community there were hundreds of actions being copied, and that is a problem!  It turned out that it wasn't filtering for is_deleted=False so there was a bunch of crap copied for each community that was not showing up and taking a super long time.

I've reinstated the cloning and improved the naming to be more convenient for community admins